### PR TITLE
fix(logger): adjust publish and ttl times to seconds

### LINKED ISSUE #3853: deis-logger is eating whole cpu
3-node cluster, fresh 1.7.0 with 1.6.1 publisher (due to issue with 404's).

One node where I've deis-logger, logger and etcd process together are eating whole cpu. If I stop deis-logger.service, etcd cpu usage is dropping. journalctl -f all together on all nodes combined is showing on average maybe <100 lines per minute.

journalctl -f -u for etcd or deis-logger doesn't show any warnings/errors.

Strace on etcd process is showing lots of GET requests to '/v2/keys/deis/logs/drain?con...', but if I remove log drain it doesn't change much in case of cpu usage.

Any advice where I should look?

### PR DESCRIPTION
**Pull request by mboersma**
Closes #3853.

### CODE REVIEW

File: logger/main.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func main() {
 			if resp != nil && resp.Node != nil {
 				drainChan <- resp.Node.Value
 			}
-			time.Sleep(time.Duration(publishInterval))
+			time.Sleep(time.Duration(publishInterval) * time.Seconds)
```
Comment (aledbf) [inline logger/main.go]: time.Sleep(publishInterval * time.Seconds) is enough, no need to use time.Duration

**Comment by mackwic** [comment]
Very slightly related but I don't know if relevant: from the [Go `flag` reference](http://golang.org/pkg/flag/#pkg-overview), the correct way to pass command-line argument is:  `/bin/logger -flag=value`.

But, the [Dockerfile use the `--flag` syntax](https://github.com/deis/deis/blob/master/logger/image/Dockerfile#L4). Is it an error ?

**Comment by bacongobbler** [comment]
@mackwic both are fine. From the same page:

> One or two minus signs may be used; they are equivalent.

**Comment by bacongobbler** [comment]
@aledbf's comment looks like a good change to implement, otherwise LGTM

**Comment by carmstrong** [comment]
LGTM.

**Comment by mboersma** [comment]
On a new `make dev-cluster` vagrant cluster from master, `top` shows the logger process to consume about 35% of CPU shares, even without any apps deployed. After this change, logger consumes a negligible (<1%) share.

File: logger/main.go (hunk 1 above)
Comment (mboersma) [inline logger/main.go]: I couldn't get the `go` compiler to pass that version. [`time.Sleep`](http://golang.org/pkg/time/#example_Sleep) expects a `Duration`, and this is the [canonical way](http://golang.org/pkg/time/#Duration) to convert an integer to seconds.

**Pull request merged**